frontend/src/metabase/query_builder/components/view/ExecutionTime/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { t } from "ttag";
+
+export const formatDuration = (time: number): string => {
```

Comment:
nit: we have other time utils in `frontend/src/metabase/lib/formatting/time.ts`

Reply:
I know but none of them could be reused here and there already is a `duration` function - I didn't want to put another function like that in there, because it'd be hard to distinguish them.

So I decided it's best to keep it local to the component it's used in.